src/Conticket/Model/Aggregates/Event/TicketPrice.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Conticket\Model\Aggregates\Event;
+
+use Money\Currency;
+use Money\Money;
+
+final class TicketPrice
+{
+
+    const CURRENCY_DEFAULT = 'BRL';
+
+    private $value;
+
+    private function __construct(Money $value)
+    {
+        $this->value = $value;
+    }
+
+    public static function fromAmount(string $amount, string $code = self::CURRENCY_DEFAULT)
+    {
+        return new self(new Money($amount, new Currency($code)));
```

Comment:
I didn't want other piece of code been client of Money class, so I receive parameters and use Money inside this class.